packages/components/src/List/Toolbar/Toolbar.scss

```diff
@@ -1,12 +1,14 @@
 // Toolbar styles
 $tc-list-toolbar-display-color: $silver !default;
-$tc-list-toolbar-background-color: $concrete !default;
+$tc-list-toolbar-background-color: $wild-sand !default;
+$tc-list-toolbar-height: 70px !default;
```

Comment:
```suggestion
$tc-list-toolbar-height: 7rem !default;
```

Reply:
Hi Jimmy,
I understand `rem` should be used here, and on <html> element, font-size is 10px.
but when I set height: 7rem it actually computes to 84px. don't know when font-size changes to 12px.

Reply:
That's weird because in the storybook, 7rem renders 70px which is correct.
If it's something in your app, it should be fixed in your app. The global font size shouldn't be changed from app to app, let the theme handle it.
Or perhaps it's an option in your browser that you change ?